audit-ci.jsonc

```diff
@@ -2,5 +2,9 @@
     // $schema provides code completion hints to IDEs.
     "$schema": "https://github.com/IBM/audit-ci/raw/main/docs/schema.json",
     "low": true,
-    "allowlist": []
+    "allowlist": [
+        // Transitive dependency in vitepress (dev-only, documentation tool).
+        // Cannot be fixed until upstream vite updates esbuild. Excluded to avoid scan noise.
```

Comment:
Please add information whether and how this vulnerability applies to our usage and why it is still accepted to ignore it.